assets/src/components/site-scan-context-provider/index.js

```diff
@@ -32,53 +33,67 @@ export const SiteScan = createContext();
 /**
  * Site Scan Actions.
  */
-const ACTION_SCANNABLE_URLS_REQUEST = 'ACTION_SCANNABLE_URLS_REQUEST';
-const ACTION_SCANNABLE_URLS_FETCH = 'ACTION_SCANNABLE_URLS_FETCH';
-const ACTION_SCANNABLE_URLS_RECEIVE = 'ACTION_SCANNABLE_URLS_RECEIVE';
-const ACTION_SCAN_INITIALIZE = 'ACTION_SCAN_INITIALIZE';
-const ACTION_SCAN_VALIDATE_URL = 'ACTION_SCAN_VALIDATE_URL';
-const ACTION_SCAN_RECEIVE_VALIDATION_ERRORS = 'ACTION_SCAN_RECEIVE_VALIDATION_ERRORS';
-const ACTION_SCAN_NEXT_URL = 'ACTION_SCAN_NEXT_URL';
-const ACTION_SCAN_CANCEL = 'ACTION_SCAN_CANCEL';
+export const ACTION_SCANNABLE_URLS_REQUEST = 'ACTION_SCANNABLE_URLS_REQUEST';
+export const ACTION_SCANNABLE_URLS_FETCH = 'ACTION_SCANNABLE_URLS_FETCH';
+export const ACTION_SCANNABLE_URLS_RECEIVE = 'ACTION_SCANNABLE_URLS_RECEIVE';
+export const ACTION_SCAN_INITIALIZE = 'ACTION_SCAN_INITIALIZE';
+export const ACTION_SCAN_URL = 'ACTION_SCAN_URL';
+export const ACTION_SCAN_RECEIVE_RESULTS = 'ACTION_SCAN_RECEIVE_RESULTS';
+export const ACTION_SCAN_COMPLETE = 'ACTION_SCAN_COMPLETE';
+export const ACTION_SCAN_CANCEL = 'ACTION_SCAN_CANCEL';
 
 /**
  * Site Scan Statuses.
  */
-const STATUS_REQUEST_SCANNABLE_URLS = 'STATUS_REQUEST_SCANNABLE_URLS';
-const STATUS_FETCHING_SCANNABLE_URLS = 'STATUS_FETCHING_SCANNABLE_URLS';
-const STATUS_READY = 'STATUS_READY';
-const STATUS_IDLE = 'STATUS_IDLE';
-const STATUS_IN_PROGRESS = 'STATUS_IN_PROGRESS';
-const STATUS_COMPLETED = 'STATUS_COMPLETED';
-const STATUS_FAILED = 'STATUS_FAILED';
-const STATUS_CANCELLED = 'STATUS_CANCELLED';
+export const STATUS_REQUEST_SCANNABLE_URLS = 'STATUS_REQUEST_SCANNABLE_URLS';
+export const STATUS_FETCHING_SCANNABLE_URLS = 'STATUS_FETCHING_SCANNABLE_URLS';
+export const STATUS_READY = 'STATUS_READY';
+export const STATUS_IDLE = 'STATUS_IDLE';
+export const STATUS_IN_PROGRESS = 'STATUS_IN_PROGRESS';
+export const STATUS_COMPLETED = 'STATUS_COMPLETED';
+export const STATUS_FAILED = 'STATUS_FAILED';
+export const STATUS_CANCELLED = 'STATUS_CANCELLED';
 
 /**
  * Initial Site Scan state.
  *
  * @type {Object}
  */
 const INITIAL_STATE = {
-	currentlyScannedUrlIndex: 0,
+	currentlyScannedUrlIndexes: [],
```

Comment:
Note: This could also be `indices` instead of `indexes`. Both are commonly used, so I have no preference.